ClientGenerator/src/googleapis/codegen/languages/csharp/default/templates/___package_name___/___package_name___.csproj.tmpl

```diff
@@ -13,8 +13,18 @@
     <RepositoryUrl>https://github.com/google/google-api-dotnet-client</RepositoryUrl>
     <PackageIconUrl>https://www.gstatic.com/images/branding/product/1x/google_developers_64dp.png</PackageIconUrl>
     <Description>
+{% noblank %}
+{% if api.module.name in features.clouds %}
+{% for old_api, cloud_api in features.clouds.items %}
+{% if old_api == api.module.name %}
+      This is not the recommended package for working with {% camel_case api.name %}, please use: {{ cloud_api }}
```

Comment:
Is the colon necessary here? It feels like it would read a little oddly.

Reply:
Removed